### PR TITLE
{data} [intel-2015b] netCDF 4.3.3.1 and netCDF-Fortran 4.4.2 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs https://github.com/hpcugent/easybuild-easyblocks/pull/734

### CODE REVIEW

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('CMake', '3.3.2'),
+    ('Bison', '3.0.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/doxygen'],
```
Comment (boegel) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: this is new, but good!

@hajgato: would you mind issuing a separate PR to add this to all Doxygen easyconfigs?

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```
Comment (boegel) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: ah, wait...

we have an easyblock for Doxygen, this seems to suggest we should update that with a version check, to use CMake for recent versions?

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: ah, hold on, this is covered by the Doxygen easyblock now, which should just be updated?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/df53f9bff4bc6736505c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d9139503e7cfe0e1bc4f for a full test report.

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb (hunk 2 above)
Comment (hajgato) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: @boegel Yes, byt that works with ConfigureMake (with a non-standard prefix, therefore needed the easyblock), but from 1.8.10 only CMake build, and it seems to work fine (meanwhile the easybock not)

**Comment by hajgato** [comment]
@boegel: ftp...

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb (hunk 2 above)
Comment (hajgato) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: @boegel: Ok, so once more. I wanted to say that should we update the easyblock, if the newer versisons work perfectly with  `CMakeMake`. The easyblock originally needed only due to a non-standard prefix-path option for `configure`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d31c20fdbfaed309d540 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9f1c84eae8bfd2df3a41 for a full test report.

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb]: OK, the tests reports I got submitted here show that it does indeed work with CMakeMake.
@hajgato: Are you up for updating the Doxygen easyblock?

**Comment by hajgato** [comment]
Needs EasyBlock PR #734 (https://github.com/hpcugent/easybuild-easyblocks/pull/734)

**Comment by boegel** [comment]
@hajgato: can you sync this with current `develop`? Easyconfigs like this got merged via another PR, I failed to notice this one...

**Comment by hajgato** [comment]
@boegel And youfailed to notice that the original PR used obsolete `cURL`....

**Comment by boegel** [comment]
@hajgato: the `cURL` is less of an issue, I'm more worried about the `HDF5`...

but I'll get this sorted, no worries

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c1a98e3b28f28311c48b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**